[BUG] pnpm dev --turbopack 编译时出现模块解析错误

## 问题描述

开发服务器无法加载google/font字体文件。

## 问题描述

在运行 `pnpm dev` 指令时，如果 package.json 文件中的 dev 脚本配置为 `"dev": "next dev --turbopack"`，会有一定概率在编译时报错，提示无法解析 `@vercel/turbopack-next/internal/font/google/font` 模块。移除 `--turbopack` 参数后可以正常编译。

## 重现步骤

1. 在 package.json 中设置 `"dev": "next dev --turbopack"`
2. 运行 `pnpm dev`
3. 等待编译过程
4. 看到模块解析错误

## 预期行为

使用 `--turbo` 参数时应该能够正常编译和启动开发服务器，提供更快的构建速度。

## 实际行为

编译过程中出现错误：
```
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
```

## 环境信息

- 操作系统: Windows
- Node.js 版本: >=20.15.0
- pnpm 版本: 10.17.1
- Next.js 版本: 15
- React 版本: 19.1.0

## 附加信息

根据网络查询，这是 Next.js 15 版本以后的一个已知 bug。目前的临时解决方案是在开发环境中去掉 `--turbopack` 参数，但这会影响开发体验和构建速度。

## 相关问题

这个问题影响了使用 Turbopack 进行快速开发的体验，建议优先修复或提供更好的解决方案。

运行 `pnpm build` 指令后产生的报错信息。

```plaintext
> Build error occurred
Error: Turbopack build failed with 6 errors:
[next]/internal/font/google/geist_a71539c9.module.css:8:9
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
   6 |   font-display: swap;
   7 |   src: url(@vercel/turbopack-next/internal/font/google/font?{%22url%22:%22https://fonts.gstatic.com/s/geist/v4/gyByhwUxId8gMEwYGFWNOITddY4.woff2%22,%22preload%22:false,%22has_size_adjust%22:true}) format('woff2');
>  8 |   unicode-range: U+0301, U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
     |         ^
   9 | }
  10 | /* latin-ext */
  11 | @font-face {

Import map: Resolved by import map


https://nextjs.org/docs/messages/module-not-found


[next]/internal/font/google/geist_a71539c9.module.css:17:9
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
  15 |   font-display: swap;
  16 |   src: url(@vercel/turbopack-next/internal/font/google/font?{%22url%22:%22https://fonts.gstatic.com/s/geist/v4/gyByhwUxId8gMEwSGFWNOITddY4.woff2%22,%22preload%22:false,%22has_size_adjust%22:true}) format('woff2');
> 17 |   unicode-range: U+0100-02BA, U+02BD-02C5, U+02C7-02CC, U+02CE-02D7, U+02DD-02FF, U+0304, U+0308, U+0329, U+1D00-1DBF, U+1E00-1E9F, U+1EF2-1EFF, U+2020, U+20A0-20AB, U+20AD-20C0, U+2113, U+2C60-2C7F, U+A720-A7FF;
     |         ^
  18 | }
  19 | /* latin */
  20 | @font-face {

Import map: Resolved by import map


https://nextjs.org/docs/messages/module-not-found


[next]/internal/font/google/geist_a71539c9.module.css:26:9
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
  24 |   font-display: swap;
  25 |   src: url(@vercel/turbopack-next/internal/font/google/font?{%22url%22:%22https://fonts.gstatic.com/s/geist/v4/gyByhwUxId8gMEwcGFWNOITd.woff2%22,%22preload%22:true,%22has_size_adjust%22:true}) format('woff2');
> 26 |   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
     |         ^
  27 | }
  28 | @font-face {
  29 |     font-family: 'Geist Fallback';

Import map: Resolved by import map


https://nextjs.org/docs/messages/module-not-found


[next]/internal/font/google/geist_mono_8d43a2aa.module.css:8:9
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
   6 |   font-display: swap;
   7 |   src: url(@vercel/turbopack-next/internal/font/google/font?{%22url%22:%22https://fonts.gstatic.com/s/geistmono/v4/or3nQ6H-1_WfwkMZI_qYFrMdmhHkjkotbA.woff2%22,%22preload%22:false,%22has_size_adjust%22:true}) format('woff2');
>  8 |   unicode-range: U+0301, U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
     |         ^
   9 | }
  10 | /* latin-ext */
  11 | @font-face {

Import map: Resolved by import map


https://nextjs.org/docs/messages/module-not-found


[next]/internal/font/google/geist_mono_8d43a2aa.module.css:17:9
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
  15 |   font-display: swap;
  16 |   src: url(@vercel/turbopack-next/internal/font/google/font?{%22url%22:%22https://fonts.gstatic.com/s/geistmono/v4/or3nQ6H-1_WfwkMZI_qYFrkdmhHkjkotbA.woff2%22,%22preload%22:false,%22has_size_adjust%22:true}) format('woff2');
> 17 |   unicode-range: U+0100-02BA, U+02BD-02C5, U+02C7-02CC, U+02CE-02D7, U+02DD-02FF, U+0304, U+0308, U+0329, U+1D00-1DBF, U+1E00-1E9F, U+1EF2-1EFF, U+2020, U+20A0-20AB, U+20AD-20C0, U+2113, U+2C60-2C7F, U+A720-A7FF;
     |         ^
  18 | }
  19 | /* latin */
  20 | @font-face {

Import map: Resolved by import map


https://nextjs.org/docs/messages/module-not-found


[next]/internal/font/google/geist_mono_8d43a2aa.module.css:26:9
Module not found: Can't resolve '@vercel/turbopack-next/internal/font/google/font'
  24 |   font-display: swap;
  25 |   src: url(@vercel/turbopack-next/internal/font/google/font?{%22url%22:%22https://fonts.gstatic.com/s/geistmono/v4/or3nQ6H-1_WfwkMZI_qYFrcdmhHkjko.woff2%22,%22preload%22:true,%22has_size_adjust%22:true}) format('woff2');
> 26 |   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
     |         ^
  27 | }
  28 | @font-face {
  29 |     font-family: 'Geist Mono Fallback';

Import map: Resolved by import map


https://nextjs.org/docs/messages/module-not-found


    at <unknown> ([next]/internal/font/google/geist_a71539c9.module.css:8:9)
    at <unknown> (https://nextjs.org/docs/messages/module-not-found)
    at <unknown> ([next]/internal/font/google/geist_a71539c9.module.css:17:9)
    at <unknown> (https://nextjs.org/docs/messages/module-not-found)
    at <unknown> ([next]/internal/font/google/geist_a71539c9.module.css:26:9)
    at <unknown> (https://nextjs.org/docs/messages/module-not-found)
    at <unknown> ([next]/internal/font/google/geist_mono_8d43a2aa.module.css:8:9)
    at <unknown> (https://nextjs.org/docs/messages/module-not-found)
    at <unknown> ([next]/internal/font/google/geist_mono_8d43a2aa.module.css:17:9)
    at <unknown> (https://nextjs.org/docs/messages/module-not-found)
    at <unknown> ([next]/internal/font/google/geist_mono_8d43a2aa.module.css:26:9)
    at <unknown> (https://nextjs.org/docs/messages/module-not-found)
 ELIFECYCLE  Command failed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -93,6 +93,7 @@
     "fumadocs-mdx": "^12.0.3",
     "fumadocs-twoslash": "^3.1.8",
     "fumadocs-ui": "^15.8.5",
+    "geist": "^1.5.1",
     "hast-util-to-jsx-runtime": "^2.3.6",
     "input-otp": "^1.4.2",
     "katex": "^0.16.25",

--- a/src/app/layout.tsx
+++ b/src/app/layout.tsx
@@ -1,19 +1,10 @@
 import { RootProvider } from "fumadocs-ui/provider";
+import { GeistMono } from "geist/font/mono";
+import { GeistSans } from "geist/font/sans";
 import type { Metadata } from "next";
-import { Geist, Geist_Mono } from "next/font/google";
 
 import { ThemeProvider } from "@/components/common/theme-provider";
 import { Toaster } from "@/components/ui/sonner";
-
-const geistSans = Geist({
-  variable: "--font-geist-sans",
-  subsets: ["latin"],
-});
-
-const geistMono = Geist_Mono({
-  variable: "--font-geist-mono",
-  subsets: ["latin"],
-});
 
 export const metadata: Metadata = {
   title: "Nomad - Your Travel Companion",
@@ -28,7 +19,7 @@ export default function RootLayout({
   return (
     <html lang="en" suppressHydrationWarning>
       <body
-        className={`${geistSans.variable} ${geistMono.variable} antialiased`}
+        className={`${GeistSans.variable} ${GeistMono.variable} antialiased`}
         suppressHydrationWarning
       >
         <ThemeProvider

